configure.in

```diff
@@ -491,6 +491,7 @@ AC_CHECK_HEADERS(libintl.h iconv.h)
 AC_CHECK_HEADERS(langinfo.h)
 AC_HEADER_TIME
 AC_TYPE_OFF_T
+AC_CHECK_SIZEOF(off_t,,)
```

Comment:
Checking the size of a type is not safe when cross-compiling, therefore we don't have these checks by purpose.

Reply:
See comment on _FILE_OFFSET_BITS. Removed this sizeof check.